utils/text_norm_whisper_like/english.py

```diff
@@ -537,6 +539,11 @@ def __init__(self, standardize_numbers=False):
         self.standardize_spellings = EnglishSpellingNormalizer()
         self.pre_standardize_spellings = EnglishSpellingNormalizer("pre_english.json")
 
+        if remove_fillers:
+            self.fillers = ['hmm', 'uh', 'ah', 'eh']  # assumes replacers have been applied
```

Comment:
what about 'oh' ? should we remove that too ?

Reply:
'oh' is quite informative. Let's keep it, which is aligned with Whisper's original filler removal.